remarkable_update_image/image.py

```
@@ -355,11 +355,18 @@ def size(self):
     def expire(self):
         self._cache.expire()
 
+    def close(self):
+        try:
+            self._archive.close()
+
+        finally:
+            return super().close()
```

Ruff (0.14.1)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)